### DISCUSSION TITLE
Plugin bundles

### DISCUSSION THREAD

**Discussion by piotrpog** [craftcms/console#72] [Plugin Store]
Are there any plans for introducing plugin bundles? 
For example, plugin developer that created google shopping feed and facebook shopping feed plugins might create bundle "shopping feeds" which costs slightly less that two plugins bought separatly.

**Comment by brandonkelly** [comment]
No plans currently but it’s something we might consider down the road.

**Comment by leevigraham** [comment]
@piotrpog Couldn't you create a new composer skeleton project that just included the two sub packages and then list that on the store?
